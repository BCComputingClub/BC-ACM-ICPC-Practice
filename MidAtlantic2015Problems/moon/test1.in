100.0
12.0
0.12
120000.0
120.0
240.0
-1.0
