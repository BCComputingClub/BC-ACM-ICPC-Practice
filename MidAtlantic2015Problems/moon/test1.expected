100.00 16.70
12.00 2.00
0.12 0.02
120000.00 20040.00
120.00 20.04
240.00 40.08
